docs/Migrating to version 2.0 guide.md

```diff
@@ -670,20 +729,77 @@ Now use `placement` instead:
 import PopoverPlacement from "@ui5/webcomponents/dist/types/PopoverPlacement.js";
 ```
 
+- `isOpen` and `close` methods are no longer present. Now `open` property can be used instead.
+
+Previously:
+```ts
+let isOpen = popover.isOpen();
+popover.close();
+```
+Now:
+```ts
+let isOpen = popover.open;
+popover.open = false;
+```
+
+- `showAt` method is no longer present.
```

Comment:
Also, if there is a replacement - please list it. For example:
Method `showAt` is no longer present. Use `opener` instead.